routes/api/users/index.js

```diff
@@ -113,10 +116,12 @@ const SendEmailConfirmation = (app, userID, email) => UsersService
     });
   });
 
+// create a local user.
 router.post('/', (req, res, next) => {
   const {
     email,
     password,
+    redirectUri,
```

Comment:
whoops, good catch. I removed that from somewhere else when I was moving things around and forgot to re-add it here.